legend-engine-test-runner-service/src/test/resources/testable/legend-testable-service-relational.pure

```diff
@@ -0,0 +1,265 @@
+###Data
+Data data::RelationalData
+{
+  Relational
+  #{
+    PersonTable:
+      {
+        columns:
```

Comment:
Should we just allow CSV format, that feels more convenient. Possibly I am missing the reason